docs/examples/stale-data.py

```diff
@@ -0,0 +1,80 @@
+"""
+Stale Data Periods
+==================
+
+Identifying stale data periods, defined as periods of
+consecutive repeating values, in time series.
```

Comment:
Same comment re: hovertext length

Reply:
Updated in 
commit 6da6f0481f795b270164db6bc61988bb2fcb6fef